src/example-relay/src/mutations/RangeAdd/LocationsForm.js

```diff
@@ -4,10 +4,11 @@
 
 import * as React from 'react';
 import { InputField, Stack, Button, Select } from '@kiwicom/orbit-components';
-import { useRelayEnvironment } from '@adeira/relay';
+import { graphql } from '@adeira/relay';
+// eslint-disable-next-line monorepo/no-internal-import
+import { useMutation } from '@adeira/relay/src/hooks';
```

Comment:
Can you export it, please? This looks like a mistake.

Reply:
Will do

Reply:
@mrtnzlml, looks ok now?